### ISSUE TITLE
User can upload same song multiple times

### ISSUE THREAD

**Issue by harmanbatheja15** [code100x/muzer#52]
### Current Behavior:

A user can upload a song as many times as he wants. And if some other user adds the same song, it again adds to the queue.

### Expected Behavior:

A user can only upload a particular song only once. And if another user uploads the same song, that song gets upvoted.

**Comment by harmanbatheja15** [comment]
@hkirat Please review this PR

**Comment by Ameerjafar** [comment]
@SujithThirumalaisamy can i work on this.

**Comment by SujithThirumalaisamy** [comment]
> @SujithThirumalaisamy can i work on this.

I think its already done. If not please raise a PR. Happy to merge it. Thankyou!
